docs/4.3/cli-docs.md

```diff
@@ -127,10 +127,13 @@ Run shell or execute a command on a remote SSH node
 | `-A, --forward-agent` | none | none | Forward agent to target node like `ssh -A`
 | `-L, --forward` | none | none | Forward localhost connections to remote server
 | `-D, --dynamic-forward ` | none | none | Forward localhost connections to remote server using SOCKS5
+| `-N, -no-remote-exec` | none | none | Don't execute remote command, useful for port forwarding
 | `--local` | none |  | Execute command on localhost after connecting to SSH node
 | `-t, --tty` | `file` |  | Allocate TTY
 | `--cluster` | none |  | Specify the cluster to connect
 | `-o, --option` | `local` |  |  OpenSSH options in the format used in the configuration file
+| `--enable-escape-sequences` | | | Enable support for SSH escape sequences. Type '~?' during an SSH session to list supported sequences.
+| `--use-local-ssh-agent` | | | Load generated SSH certificates into the local ssh-agent (specified via `$SSH_AUTH_SOCK`). You can also set `TELEPORT_USE_LOCAL_SSH_AGENT` environment variable. Default is true.
```

Comment:
```suggestion
| `--no-use-local-ssh-agent` | | | Do not load generated SSH certificates into the local ssh-agent (specified via `$SSH_AUTH_SOCK`). Useful when using `gpg-agent` or Yubikeys. You can also set the `TELEPORT_USE_LOCAL_SSH_AGENT` environment variable to `false` (default `true`)
```

How about this? `--use-local-ssh-agent` is the default, so I figure we should be explicit about how to disable it rather than enable it.

Reply:
Yeah, I like this.